### DISCUSSION TITLE
Compiling dalphaball

### DISCUSSION THREAD

**Discussion by orlyDenovai** [RosettaCommons/rosetta#167] [Build & Install]
Hi,

I'm running into an issue with compiling dalphaball from rosetta.source.release-371. When invoking make in the DAlphaball directory, I get the following type of errors:

gcc -o ./DAlphaBall.gcc -O3 DAlphaBall.o sos_minor_gmp.o alf_tools_gmp.o binding.o adjust.o alfcx.o alfcx_tools.o cavballs.o delcx.o truncate_real.o measure_tools.o surface_tools.o surface_only.o surface_deriv_coord.o volume_tools.o volume_only.o volume_deriv_coord.o vector.o write_simplices.o -L/usr/local/lib -lgmp  -lm -lgfortran
/usr/bin/ld: alf_tools_gmp.o:(.bss+0x1e85be0): multiple definition of `coord_gmp'; sos_minor_gmp.o:(.bss+0x1e85be0): first defined here
...
 /usr/bin/ld: alf_tools_gmp.o:(.bss+0x13b0): multiple definition of `a12_mp'; sos_minor_gmp.o:(.bss+0x13b0): first defined here
/usr/bin/ld: alf_tools_gmp.o:(.bss+0x13c0): multiple definition of `a11_mp'; sos_minor_gmp.o:(.bss+0x13c0): first defined here

I'm working on WSL2, using gfortran 12.2.0

Any suggestions for how to resolve this issue will be highly appreciated.
Thanks in advance,
Orly

**Comment by roccomoretti** [comment]
I believe that this is the same issue which was fixed in PR #46 -- That's post 3.14, so you may need to download the latest Github version to get it. (Or just replace the three files references in that PR -- DAlphaball is pretty self-contained.)
